model/generate.go

```diff
@@ -685,7 +685,7 @@ func (g *GeneratedProject) validateNoRedefine(cachedProject projectMaps) error {
 func isNonZeroBV(bv parserBV) bool {
 	if bv.DisplayName != "" || len(bv.Expansions) > 0 || len(bv.Modules) > 0 ||
 		bv.Disable || len(bv.Tags) > 0 ||
-		bv.BatchTime != nil || bv.Activate != nil || bv.PatchOnly != nil || bv.Stepback != nil || len(bv.RunOn) > 0 {
+		bv.BatchTime != nil || bv.PatchOnly != nil || bv.Stepback != nil || len(bv.RunOn) > 0 {
```

Comment:
@ablack12 I originally changed this to include `bv.Activate != nil` since it seems like we wouldn't allow overwriting the build variant level activate when it's already been set, but some of the generate tasks tests seem to be confirming that the overwriting is allowed. Is it intentional that you can overwrite an existing build variant activate setting?

Reply:
I'm going to merge this as-is, but I'd still be interested to know if activate can be overrridden. If so, I can update the docs in a follow-up PR to mention that activate is a special case field that _can_ be overridden in generate.tasks.